modules/virt-planning-bare-metal-cluster-for-ocp-virt.adoc

```diff
@@ -9,13 +9,13 @@
 
 If you will use {VirtProductName}, it is important to be aware of several requirements before you install your bare metal cluster.
 
-* If you want to use live migration features, you must have multiple worker nodes _at the time of cluster installation_. This is because live migration requires the cluster-level high availability (HA) flag to be set to true. The HA flag is set when a cluster is installed and cannot be changed afterwards. If there are fewer than two worker nodes defined when you install your cluster, the HA flag is set to false for the life of the cluster.
+* If you want to use high availability (HA) features, you must have multiple worker nodes _at the time of cluster installation_. This is because live migration requires the cluster-level high availability flag to be set to true. The HA flag is set when a cluster is installed and cannot be changed afterwards. If there are fewer than two worker nodes defined when you install your cluster, the HA flag is set to false for the life of the cluster.
```

Comment:
```suggestion
* If you want to use live migration features, you must have multiple worker nodes _at the time of cluster installation_. This is because live migration requires the cluster-level `controlPlaneTopology` set to be `HighlyAvailable` . The `controlPlaneTopology` is set when a cluster is installed and cannot be changed afterwards. If there are fewer than two worker nodes defined when you install your cluster, the `controlPlaneTopology` is set to TODO for the life of the cluster.
```

Reply:
If you want to use high availability (HA) features, you must have multiple control-plane,master and worker nodes _at the time of cluster installation_. Also, live migration requires the cluster-level `controlPlaneTopology` set to be `HighlyAvailable` . The `controlPlaneTopology` is set when a cluster is installed and cannot be changed afterwards. If there are fewer than two worker nodes defined when you install your cluster, the `controlPlaneTopology` is set to `SingleReplica` for the life of the cluster.